### PR TITLE
Mui login form

### PR DESCRIPTION
**Pull request by ozkalai**
Test me! 'MUI'
[Link to MUI-LOGIN-FORM](https://mui-log-refine.pankod.com)

Hey, I'm client! Test me! 'MUI'
[Link to MUI-LOGIN-FORM](https://mui-log-client.refine.pankod.com)

Test me! 'MUI'
[Link to MUI-LOGIN-FORM](https://mui-log-refine.pankod.com)

Hey, I'm client! Test me! 'MUI'
[Link to MUI-LOGIN-FORM](https://mui-log-client.refine.pankod.com)

Test me! 'MASTER'
[Link to MUI-LOGIN-FORM](https://mui-log-refine.pankod.com)

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

**Test plan (required)**

Demonstrate the code is solid. If not, please add `WIP:` in its title.

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: packages/mui/src/components/pages/login/index.tsx [hunk 1]
```diff
@@ -12,36 +12,36 @@ import {
...
+        handleSubmit,
+        formState: { errors },
+    } = useForm<ILoginForm>();
+
     const { mutate: login, isLoading } = useLogin<ILoginForm>();
     const translate = useTranslate();
 
-    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
-        event.preventDefault();
+    const onSubmit: any = (data: ILoginForm) => login(data);
```
Comment (omeraplak) [inline packages/mui/src/components/pages/login/index.tsx:38]: let's specify the type

File: examples/base/mui/src/App.tsx [hunk 2]
```diff
@@ -21,62 +25,81 @@ import { UseModalFormList } from "pages/useModalForm";
...
+        if (currentHour >= 19 || currentHour <= 6) {
+            setTheme("dark");
+        } else {
+            setTheme("light");
+        }
+    };
+
+    useEffect(() => {
+        setThemePreference();
+    }, []);
```
Comment (salihozdemir) [inline examples/base/mui/src/App.tsx:43]: [Mui System Preference](https://mui.com/material-ui/customization/dark-mode/#system-preference)

You can detect a theme by users preference like chrome or OS theme selection.

File: examples/base/mui/src/index.tsx [hunk 3]
```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
+import { StylesProvider } from "@material-ui/styles";
+import "@pankod/refine-mui/dist/styles.min.css";
 
 ReactDOM.render(
     <React.StrictMode>
-        <App />
+        <StylesProvider injectFirst>
```
Comment (salihozdemir) [inline examples/base/mui/src/index.tsx:9]: Is `StylesProvider` the legacy styling solution for MUI isn't it?

File: packages/mui/src/components/pages/login/index.tsx [hunk 4]
```diff
@@ -74,26 +73,48 @@ export const LoginPage: React.FC = () => {
...
+                                    variant="h4"
                                     align="center"
-                                    color="#626262"
+                                    sx={{
+                                        fontWeight: "700",
+                                        fontSize: "30px",
+                                        letterSpacing: "-0.04px",
+                                        lineHeight: "38px",
+                                        color: palette.text.secondary,
+                                    }}
```
Comment (salihozdemir) [inline packages/mui/src/components/pages/login/index.tsx:86]:
```suggestion
                                    sx={{
                                        fontWeight: "700",
                                        fontSize: "30px",
                                        letterSpacing: "-0.04px",
                                        lineHeight: "38px",
                                        color: text.secondary,
                                    }}
```

You don't have to use `useTheme` for getting the palette. You can see details in this [link](https://mui.com/system/basics/#shorthands)

**Review by omeraplak — APPROVED** [review]

File: examples/base/mui/src/index.tsx [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
+import "@pankod/refine-mui/dist/styles.min.css";
```
Comment (omeraplak) [inline examples/base/mui/src/index.tsx:4]: we can move this to app.tsx

**Review by omeraplak — CHANGES_REQUESTED** [review]

File: examples/base/mui/src/App.tsx [hunk 6]
```diff
@@ -18,65 +23,75 @@ import {
     UseStepsFormEdit,
 } from "pages/useStepsForm";
 import { UseModalFormList } from "pages/useModalForm";
+import "@pankod/refine-mui/dist/styles.min.css";
```
Comment (omeraplak) [inline examples/base/mui/src/App.tsx:26]: The import order should be like this,
https://github.com/pankod/refine/blob/master/examples/base/src/App.tsx#L9

Comment (omeraplak) [inline examples/base/mui/src/App.tsx:26]: The import order should be like this,
https://github.com/pankod/refine/blob/master/examples/base/src/App.tsx#L9

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Review by salihozdemir — APPROVED** [review]

File: packages/mui/tsup.config.ts [hunk 7]
```diff
@@ -1,5 +1,6 @@
 import { defineConfig } from "tsup";
 import { NodeResolvePlugin } from "@esbuild-plugins/node-resolve";
+import copyStaticFiles from "esbuild-copy-static-files";
```
Comment (omeraplak) [inline packages/mui/tsup.config.ts:3]:
```suggestion
```

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Pull request merged**